apps/fabric-website/package.json

```diff
@@ -38,8 +38,10 @@
     "write-file-webpack-plugin": "^4.1.0"
   },
   "dependencies": {
+
```

Comment:
when you add a dependency, you should run `npm run generate` from the root to regen the shrinkwrap.